common/saber_base.h

```diff
@@ -38,6 +38,13 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(POWERSAVE)                    \
     DEFINE_EFFECT(BATTERY_LEVEL)                \
     DEFINE_EFFECT(FAST_ON)                      \
+    DEFINE_EFFECT(SECONDARY_IGNITION)		\
+    DEFINE_EFFECT(SECONDARY_RETRACTION)		\
+    DEFINE_EFFECT(QUOTE)			\
+    /* Triggers the change for sets of sounds within the font from one alternative to another */                \
+    DEFINE_EFFECT(ALT_SOUND)			\
+    /* Optional sound used during Transition Effect or TransitionPulse */         \
```

Comment:
If this effect is meant to be triggered by the style, then I think it might be a good idea to call that out.

Reply:
added